docs/install/install.rst

```diff
@@ -0,0 +1,49 @@
+.. meta::
+  :description: TransferBench documentation 
+  :keywords: TransferBench, API, ROCm, HIP
+
+---------------------------
+TransferBench Installation
+---------------------------
+
```

Comment:
Shouldn't there be an intro sentence before these two bullets?
Perhaps system requirements?

Reply:
We need the content from the Component team.